guides/common/modules/proc_configuring-hammer-cli-to-accept-freeipa-credentials.adoc

```diff
@@ -0,0 +1,32 @@
+[id="configuring-hammer-cli-to-accept-{FreeIPA-context}-credentials_{context}"]
+= Configuring Hammer CLI to accept {FreeIPA} credentials
+
+Configure the {Project} Hammer CLI tool to use {FreeIPA} to authenticate users.
+
+.Prerequisites
+* You have enabled {FreeIPA} access to the {Project} API.
+See xref:Configuring_FreeIPA_Authentication_on_Server_{context}[].
+
+.Procedure
+
+. Open the `~/.hammer/cli.modules.d/foreman.yml` file on your {ProjectServer} and update the list of `foreman` parameters:
```

Comment:
```suggestion
. Open the `~/.hammer/cli.modules.d/foreman.yml` file on your {ProjectServer} and update the list of `foreman` parameters.
```

Reply:
I'll stick to what the IBM guide recommends.